app/src/main/res/menu/menu_profile_fragment.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/miLogOut"
```

Comment:
Just a small question to you what does the "mi" abbreviation mean?

Reply:
Menu Item